nuxeo-web-ui-ftest/webdriver/test/features/step_definitions/tasks.js

```diff
@@ -2,4 +2,7 @@
 
 module.exports = function () {
   this.Then('I can see the list of tasks', () => this.ui.drawer.tasks.isVisible().should.be.true);
+  this.Then('I can see the View Tasks Dashboard link', () => this.ui.tasksLink.isVisible().should.be.true);
```

Comment:
To be more consistent with the app model, it should rather be something like `this.ui.drawer.tasksLink.isVisible().should.be.true`